awx/main/scheduler/task_manager.py

```diff
@@ -233,6 +235,7 @@ def start_task(self, task, rampart_group, dependent_tasks=None, instance=None):
         else:
             if type(task) is WorkflowJob:
                 task.status = 'running'
+                task.send_notification_templates('running')
```

Comment:
Don't you need the same kind of `connection.on_commit(lambda: workflow_job.send_notification_templates('running'))` here as well?

Also, I can't understand why we have an exception of "succeeded" (as opposed to "successful") for a finished job, but pass in "running" for the on-start case, since the user-facing terminology is "started"